graphql/schema.graphql

```diff
@@ -703,7 +703,7 @@ type Mutation {
     email: String
     isNewsletterOptIn: Boolean
     fields: NewsletterFieldsInput
-  ): Boolean @cacheControl(scope: PRIVATE)
+  ): Boolean @withCurrentProfile @cacheControl(scope: PRIVATE)
```

Comment:
This mutation is deprecated...

Reply:
Where I can find this info? At the code, it isn't.